components/dashboard/src/components/InputWithCopy.tsx

```diff
@@ -31,12 +31,11 @@ export const InputWithCopy: FC<Props> = ({ value, tip = "Click to copy", classNa
     return (
         // max-w-lg is to mirror width of TextInput so Tooltip is positioned correctly
         <div className={`w-full relative max-w-lg ${className ?? ""}`}>
-            <TextInput value={value} disabled className="w-full pr-8 overscoll-none" />
+            <TextInput value={value} disabled className="w-full pr-8 overscroll-none" />
 
             <Tooltip content={tip} className="absolute top-1.5 right-1">
```

Comment:
I ended up giving up on a Tooltip component for this PR. We cannot use it universally because on places like here, where we position the origin absolutely, it simply misbehaves. Keeping it in the commit history so that we can bring it back, though.